frame/benchmarking/src/lib.rs

```diff
@@ -69,6 +69,9 @@ pub use paste;
 /// (or not) by each arm. Syntax is available to allow for only the range to be drawn upon if
 /// desired, allowing an alternative instancing expression to be given.
 ///
+/// Note that the ranges are *inclusive* on both sides. This is in contrast to ranges in Rust which
```

Comment:
As a follow up, we can just change the macro syntax to `MIN ..= MAX`, which I also think solve this syntax mismatch right?